pkg/apis/serving/fieldmask_test.go

```diff
@@ -51,6 +51,68 @@ func TestVolumeMask(t *testing.T) {
 	}
 }
 
+func TestCapabilitiesMask_SecureEnabled(t *testing.T) {
```

Comment:
```suggestion
func TestCapabilitiesMask_SecurePodDefaultsEnabled(t *testing.T) {
```

Reply:
Sounds good. I used `SecureEnabled` based on what the [PR](https://github.com/knative/serving/pull/13398/files#diff-b77e64792c940b2c78ffffeef2af115babdd7fa455a1714e4478cec9c9c69593) that introduced secure-pod-defaults did. I've replaced both secure-pod-defaults related tests to use `SecurePodDefaultsEnabled` instead of `SecureEnabled`